samples/README_TEMPLATE.md

```diff
@@ -0,0 +1,135 @@
+======================================
+
+> REQUIRED
+> 
+> Brief description of the sample.
+> 
+> Example:
+
+This example demonstrates the usage of the I/O pins API by giving an example
+of how to initialize and manage the status of an I/O pin.
+
+The example toggles the status of an LED every second.
+
+Requirements
+------------
+
+> REQUIRED
+> 
+> This section lists the hardware and software components needed to run the
+> sample application. Computer and PyCharm components are omitted.
+>
+> Example: 
+
+To run this example you need:
+
+* One XBee3 radio module with MicroPython support.
+* One carrier board for the radio module (XBIB-U-DEV or XBIB-C board).
+* One standalone HDC1080 humidity and temperature sensor (not necessary if you
+  are using an XBIB-C carrier board).
+
+Setup
+-----
+
+> REQUIRED
+> 
+> Describe here all the steps needed to connect and configure the hardware
+> needed by the sample as well as any software configuration or changes in
+> the code required prior to run the sample.
+>
+> Example:
+
+Make sure the hardware is set up correctly:
+
+1. Plug the XBee3 radio module into the XBee adapter and connect it to your
+   computer's USB port.
+
+Run
+---
+
+> REQUIRED
+> 
+> In this section you must describe the steps needed to launch the sample and
+> verify it is working properly.
+>
+> Example:
+
+The example is already configured, so all you need to do is to compile and
+launch the application.
+
+Verify the LED corresponding to the configured pin (`ON/SLEEP/DIO9`) in the
+carrier board starts blinking (toggles its status every second). Every time
+the LED toggles, the application indicates it through the XBee REPL console:
+
+    - LED ON
+    - LED OFF
+    - LED ON
+    - LED OFF
+
+Required libraries
+--------------------
+
+> OPTIONAL
+>
+> This section is used by the **XBee MicroPython plugin for PyCharm** to import
+> the libraries required by the sample application once the sample is imported.
+> The content of the section is a list with the ID of required libraries from
+> the `lib` directory. ID must equal the name of the library folder.
+> 
+> You can remove this section if the sample does not require any library.
+> 
+> Example:
+
+* uftp
+
+Supported platforms
+-------------------
+
+> REQUIRED
+> 
+> This section contains a list with the ID of XBee platforms compatible with
+> the sample and the minimum firmware required by each one. If you don't know
+> what's the minimum firmware version required, leave it with the default one
+> listed here. You can take the ID of the platforms from the
+> [platforms definition file](../platforms/platforms.xml).
+> 
+> Example:
+   
+* Digi XBee3 Zigbee 3 - minimum firmware version: 1006
+* Digi XBee3 802.15.4 - minimum firmware version: 2003
+* Digi XBee3 DigiMesh 2.4 - minimum firmware version: 3002
+* Digi XBee3 Cellular LTE-M/NB-IoT - minimum firmware version: 11410
+* Digi XBee3 Cellular LTE CAT 1 - minimum firmware version: 31010
```

Comment:
Once "XBee3 Cellular LTE Cat 1 Verizon" is available, this name isn't going to make much sense. Especially since the VR values won't be the same as LTE Cat 1 "AT&T".

Reply:
Should we rename **Digi XBee3 Cellular LTE CAT 1** to **Digi XBee3 Cellular LTE CAT 1 AT&T** and add the **XBee3 Cellular LTE CAT 1 Verizon** platform when it is available?

Reply:
I'm not sure if "LTE Cat 1 AT&T" is the best name to use, unless the module is locked to AT&T. (I'm not sure whether it is.) In any case, currently we just use the name "... LTE Cat 1" in documentation and other material.

We might need to ask someone like Kyle Sporre what he thinks.

Reply:
At the moment the official name of the product in the web page is **Digi XBee3 Cellular LTE CAT 1**, do you think the name will change once the **Digi XBee3 Cellular LTE Cat 1 Verizon** is out?

Reply:
Per our e-mail conversation, the name of the product keeps with Digi XBee3 Cellular LTE CAT 1 (well actually I changed it to Digi XBee3 Cellular LTE Cat 1)